fairtally/cli.py

```diff
@@ -0,0 +1,76 @@
+import io
+import json
+from pathlib import Path
+import click
+from howfairis import Checker
+from howfairis import Compliance
+from howfairis import Repo
+from jinja2 import Template
+from tqdm import tqdm
+from fairtally.get_badge_color import get_badge_color
+from fairtally.redirect_stdout_stderr import RedirectStdStreams
+
+
+@click.command()
+@click.argument("urls", nargs=-1)
+@click.option("--html", "output_file_html",
+              help="Filename of where to write the results as HTML.",
+              default=None, type=click.File("wt"))
+@click.option("--json", "output_file_json",
+              help="Filename of where to write the results as JSON.",
+              default=None, type=click.File("wt"))
+def cli(urls=None, output_file_html=None, output_file_json=None):
+
+    def write_as_json():
+        with output_file_json:
+            json.dump(results, output_file_json, sort_keys=True)
+
+    def write_as_html():
+        parent = Path(__file__).parent
+        template_file = parent / "data" / "index.html.template"
+        with open(template_file) as f:
+            template = Template(f.read())
+        s = template.render(results=results)
+        with output_file_html:
+            output_file_html.write(s)
+
+    if urls is None:
```

Comment:
Urls var is empty set when no urls are given.
```suggestion
    if len(urls) == 0:
```

Reply:
Indeed you're right